Tribler/LICENSE.txt

```diff
@@ -1,679 +0,0 @@
-﻿------------------------------------------------------------------------------
-
-    Tribler content-delivery library.
-
-    Development of the Tribler library was supported by numerous research 
-    grants:
-     - European Institute of Innovation and Technology (EIT) in the project for
-       Seamless P2P Video Streaming under grant TFMC 12199.
-     - European Community's Seventh Framework Programme in the QLECTIVES project
-       under contract no 231200.
-     - European Community's Seventh Framework Programme in the P2P-NEXT project
-       under contract no 216217 (see ns-LICENSE.txt for details).
-     - BSIK Freeband Communication I-Share project (Dutch Ministry of Economic 
-       Affairs)
-     - Netherlands Organisation for Scientific Research (NWO) grant 612.060.215
-     - Dutch Technology Foundation STW: Veni project DTC.7299
-     - European Community's Sixth Framework Programme in the P2P-FUSION project
-       under contract no 035249.
- 
```

Comment:
We used to list our grants. This is more of a pointer and courtesy. Can be removed.